### PR TITLE
Run console tests in less parallel process due to SHM issue and timeout issue

### PR DESCRIPTION
**Pull request by liangxia**
console tests in container need SHM to be 2GB at least.
If we run them in parallel, then 2GB are shared across those parallel process, which is much smaller then requirement, cause random failures for console tests.

So in this PR, move console tests out and run them in serial.

/cc @yapei @yanpzhan @zxiao-redhat @jhou1

### CODE REVIEW

File: ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh [hunk 1]
```diff
@@ -35,19 +35,22 @@ export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/parallel/normal"
...
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and @admin and not @serial and not @console and ${E2E_SKIP_TAGS}\" -p junit"' || true
 
-# run the rest tests in serial
-export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/serial"
 export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS="${USERS}"
+# run serial tests
+export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/serial"
 cucumber --tags "${E2E_RUN_TAGS} and @serial and ${E2E_SKIP_TAGS}" -p junit || true
+# run console tests
+export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/console"
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh:52]: Is it ok to also put console junit results in the `${ARTIFACT_DIR}/serial` directory?

Comment (liangxia) [reply ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh:52]: Fine to me. Let's wait for the tests to finish then I'll make it this way.

**Comment by yapei** [comment]
ack, thanks for letting us know the changes

**Comment by liangxia** [comment]
Unfortunately, running console tests in serial can not finish in the 8 hour timeout.

@jhou1 @yapei Thoughts ?

**Comment by liangxia** [comment]
Currently we run 4 parallel process for all tests (including console tests), except serial tests.

I'll try to increase existing parallel process from 4 to 6 for non-console tests.
And add a new stage for console tests with 2 parallel  process, given we have random console failures for 4 parallel process and running console in serial can not finish in the 8 hours timeout.

**Comment by liangxia** [comment]
PR ready for review.

With 28 of 339 tests failed, and most of them are due to new changes to oc apis. The error is similar to `error: unable to load template file "ocp12041-s2i.json": resource mapping not found for name: "ruby-helloworld-sample" namespace: "" from "ocp12041-s2i.json": no matches for kind "Template" in version "v1"`. I'll work with feature team to bump up the api versions in their test data files.

**Comment by liangxia** [comment]
The PR is trying to reduce some random failures as below,
```
Message:

    chrome not reachable
  (Session info: chrome=100.0.4896.127) (Selenium::WebDriver::Error::WebDriverError)
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-verification-tests-master-nightly-4.11-e2e-aws-ipi/1518854395045875712

**Comment by jhou1** [comment]
@yapei Can you please give suggestions about how console tests should be executed? Running in a serial reduces the number of failures, but the duration of the test is not ideal; Running in parallel saves time and cluster resources, but has some unexpected test flakes.

**Comment by jhou1** [comment]
Another suggestion would be having a dedicated job that runs only console tests in a less frequent manner (once every week?)

**Comment by liangxia** [comment]
Any updates for this ?

**Comment by jhou1** [comment]
Thank you.

/lgtm
/approve

**Pull request merged**